lib/auth/auth.go

```diff
@@ -4264,6 +4279,13 @@ func (a *Server) deleteUnusedKeys(ctx context.Context) error {
 	return trace.Wrap(a.keyStore.DeleteUnusedKeys(ctx, usedKeys))
 }
 
+func (a *Server) GetLicense(ctx context.Context, req *proto.SubmitUsageEventRequest) (string, error) {
+	if a.license == nil {
+		return "", trace.NotFound("license not found")
+	}
+	return fmt.Sprintf("%s%s", a.license.CertPEM, a.license.KeyPEM), nil
```

Comment:
Users will be able to download their license. This will require a permission that only `editor` will have by default.

> Why not embed the CertPem and KeyPem into a struct and return it instead of combining the certPEM and keyPEM ?

Could be. I just did it this way because this is how it's going to be used in the web UI, which is the only use case we have for returning the license. 

Do you think we should return the license in a struct here and format it in the proxy endpoint?

Reply:
It depends how the returned licence will be used so if an end user will need to have combined cert and key that is fine.